python/example_code/kinesis/README.md

```diff
@@ -94,8 +93,7 @@ in the `python` folder.
 
 * [Kinesis Developer Guide](https://docs.aws.amazon.com/streams/latest/dev/introduction.html)
 * [Kinesis API Reference](https://docs.aws.amazon.com/kinesis/latest/APIReference/Welcome.html)
-* [SDK for Python Kinesis reference](https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/kinesis.html)
-* [SDK for Python Kinesis Data Analytics reference](https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/kinesisanalyticsv2.html)
```

Comment:
Was this link removed intentionally?

Reply:
I think you're right we should keep it, so I've added it back.